lib/blue_apron/spree_client.rb

```diff
@@ -236,7 +236,7 @@ def perform_http_action(verb, url, options = {})
       end
 
       def setup_timeouts(request)
-        #request.options.timeout = 10
```

Comment:
It was initially commented out... I uncommented it and assigned to the same value that `open_timeout` is already getting.